drivers/clock_control/clock_control_mcux_ccm.c

```diff
@@ -108,6 +108,9 @@ static int mcux_ccm_get_subsys_rate(const struct device *dev,
 				(CLOCK_GetRootPreDivider(kCLOCK_RootUart4)) /
 				(CLOCK_GetRootPostDivider(kCLOCK_RootUart4)) /
 				10;
+		if(CONFIG_SOC_SERIES_IMX8MN_M7){
+			*rate=24000000;			//24MHz oscillator is used for CC8MNano DVK
```

Comment:
Is the rate always 24Mhz for IMX8 nano?

Reply:
Yes. We ported Zephyr for a low power use case scenario. Therefore, we used a clock source which:

1. Saves power due to lower clock rate.
2. Runs independently even when Linux/A53 cores are sleeping.